_lzbench/lzbench.cpp

```diff
@@ -744,6 +745,30 @@ void usage(lzbench_params_t* params)
     fprintf(stderr,"  " PROGNAME " -t0u0i3j5 -ezstd fname = the same as above with aggregated parameters\n");
 }
 
+char* cpu_brand_string(void)
+{
+    uint32_t mx[4], i, a, d;
+    //static char cpu_brand_string[3*sizeof(mx)+1];
+
+    char* cpu_brand_string = (char*)malloc(3*sizeof(mx)+1);
```

Comment:
cpu_brand_string is malloc but never free.

Reply:
I know it's not. It's used to printf().
I have corrected it.